### ISSUE TITLE
Directionality of ankle pain

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11077]
**Preferred term label:**
Lateral ankle pain
Medial ankle pain
Posterior ankle pain
Anterior ankle pain

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030840

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
All we have so far is Ankle sprain but I expect we will add more

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001320
- label: Lateral ankle pain
- parents: [Ankle pain (HP:0030840)](https://hpo.jax.org/app/browse/term/HP:0030840)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the outer side of the ankle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11077
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001321
- label: Medial ankle pain
- parents: [Ankle pain (HP:0030840)](https://hpo.jax.org/app/browse/term/HP:0030840)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the inner side of the ankle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11077
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001322
- label: Posterior ankle pain
- parents: [Ankle pain (HP:0030840)](https://hpo.jax.org/app/browse/term/HP:0030840)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the back of the ankle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11077
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001323
- label: Anterior ankle pain
- parents: [Ankle pain (HP:0030840)](https://hpo.jax.org/app/browse/term/HP:0030840)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the front of the ankle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11077
- synonyms:
